addons/base_address_extended/models/base_address_extended.py

```diff
@@ -85,4 +86,5 @@ def _split_street(self):
             vals[field_name] = street_raw.rstrip(street_format[previous_pos:])
             #assign the values to the fields. Note that a write(vals) would cause a recursion since it would bypass the cache
             for k, v in vals.items():
-                setattr(partner, k, v)
+                if v is not False:
+                    setattr(partner, k, v)
```

Comment:
is to avoid explicit text 'False' in address, like this example:
![captura de pantalla de 2016-12-23 19-12-48](https://cloud.githubusercontent.com/assets/5335402/21464493/5be8bcae-c944-11e6-9a14-696c3f12c05f.png)

Although it should be:
``` python
setattr(partner, k, v or '')
```